paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/AddPaymentMethodRequirement.kt

```diff
@@ -0,0 +1,56 @@
+package com.stripe.android.lpmfoundations.paymentmethod
+
+import com.stripe.android.model.PaymentIntent
+import com.stripe.android.model.PaymentMethod
+
+internal enum class AddPaymentMethodRequirement {
+    /** A special case that indicates the payment method is always unsupported by PaymentSheet. */
+    Unsupported {
+        override fun meetsRequirements(metadata: PaymentMethodMetadata): Boolean = false
+    },
```

Comment:
This will allow us to implement payment method definitions without enabling them correct?